Sources/Overload/OvRendering/include/OvRendering/Settings/EMaterialDomain.h

```diff
@@ -0,0 +1,21 @@
+/**
+* @project: Overload
+* @author: Overload Tech.
+* @licence: MIT
+*/
+
+#pragma once
+
+#include <cstdint>
+
+namespace OvRendering::Settings
+{
+	/**
+	* Material Domains
```

Comment:
We should add a note somewhere where that we can only support 2 values given the pipeline state layout (single bit for domain)